node/out_response.js

```diff
@@ -218,10 +218,24 @@ TChannelOutResponse.prototype.sendError = function sendError(codeString, message
 TChannelOutResponse.prototype.emitFinish = function emitFinish() {
     var self = this;
 
-    if (!self.end) {
-        self.end = self.timers.now();
+    if (self.end) {
+        self.logger.warn('out response double emitFinish', {
+            end: self.end,
+            now: Date.now(),
+            serviceName: self.inreq.serviceName,
+            cn: self.inreq.headers.cn,
+            endpoint: String(self.inreq.arg1),
```

Comment:
yes lets use `endpoint` everywhere. Just fitting with local style because what even is consistency